Tasks/UnityActivateLicense/UnityActivateLicenseV1/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "Dinomite Studios",
   "license": "MIT",
   "dependencies": {
-    "@types/node": "^18.0.0",
+    "@types/node": "^20.0.0",
```

Comment:
I am just a bystanding with just a little knowledge as the next person wanting to find a fix for this :P , but shouldn't this version be "@types/node": "^20.3.1"
according to the migration documentation?
https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode20.md